batch/deployment.yaml

```diff
@@ -42,13 +42,10 @@ spec:
          - name: batch-user-secret
            mountPath: /batch-user-secret
            readOnly: true
-         - name: jwt-secret
-           mountPath: /jwt-secret
+         - name: jwt-secret-key
+           mountPath: /jwt-secret-key
            readOnly: true
-         - name: gsa-key
-           mountPath: /batch-gsa-key
-           readOnly: true
-         - name: jwt
+         - name: batch-jwt
```

Comment:
What happened to the gsa-key? Did you remove it because it's not used?

Reply:
I incorrectly deleted it. I put it back.  If we're deploy, I use the original key, otherwise I use test-gsa-key.